frontend/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import {
+  Link
+} from "react-router-dom";
+import { createStyles, Theme, makeStyles } from "@material-ui/core/styles";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import PeopleIcon from "@material-ui/icons/People";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+import AppsIcon from "@material-ui/icons/Apps";
+import AssignmentIcon from "@material-ui/icons/Assignment";
+import EmojiObjectsIcon from "@material-ui/icons/EmojiObjects";
+import SettingsIcon from "@material-ui/icons/Settings";
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+
+
+export interface MenuProps {}
+
+const Menu: React.FC<MenuProps> = (props) => {
+  const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+      root: {
+        width: "23%",
+        maxWidth: 360,
+        minWidth: 180,
+        backgroundColor: theme.palette.background.default,
+        borderRight: "1px solid #666",
+        color: "#fff",
+        "& .MuiListItem-root": {
+          "&:hover": {
+            color: "#1A90FF",
+            backgroundColor: "#1C1C1C",
+          },
+        },
+        "& .MuiListItemIcon-root": {
+          color: "inherit",
+        },
+      },
+      span: {
+        paddingLeft: "13px",
+        float: "left",
+        color: "#9E9E9E",
+        fontSize: "14px",
+      },
+      userDiv:{
+        borderBottom: "1px solid #666666"
+      }
+    })
+  );
+
+  interface ListItemLinkProps {
+    path: string;
+    icon: React.ReactNode;
+    text: string;
+  }
+
+  const classes = useStyles();
+
+  const ListItemLink = ({ path, icon, text }: ListItemLinkProps) => {
+    return (
+      <ListItem button component={Link} to={path}>
+        <ListItemIcon>{icon}</ListItemIcon>
+        <ListItemText primary={text} />
+      </ListItem>
+    );
+  };
```

Comment:
I think definition of ListItemLink component shouldn't be inside Menu component.

Reply:
ok, I have created new folder for this component.